python/TestHarness/util.py

```diff
@@ -15,7 +15,13 @@
 import yaml
 import sys
 
-TERM_COLS = int(os.getenv('MOOSE_TERM_COLS', '110'))
+DEFAULT_TERM_COLS = None
+try:
+    DEFAULT_TERM_COLS = os.get_terminal_size().columns * 5/6
+except:
+    DEFAULT_TERM_COLS = 110
+    pass
+TERM_COLS = int(os.getenv('MOOSE_TERM_COLS', DEFAULT_TERM_COLS))
```

Comment:
I am fine with this. Lets see what other people have to say!

_I tried to change this a while ago and received "no" feedback_